driver-core/src/main/java/com/datastax/driver/core/policies/ErrorAwarePolicy.java

```diff
@@ -102,7 +101,8 @@ protected Host computeNext() {
                         return host;
                     }
 
-                    if (currentTime - excludedTime > retryPeriodMillis) {
+                    if (clock.getTime() - excludedTime > retryPeriodMillis) {
```

Comment:
Re-inlined the call to the clock to optimize the case where `excludedTime == null` (which should be the most frequent now).